.github/workflows/release-prod.yml

```diff
@@ -10,6 +10,11 @@ on:
 jobs:
   release_master:
     name: Tag, Release and Publish
```

Comment:
I think we could combine `release-prod.yml` and `publish-qa.yml` into a single workflow by building on branch pushes. The `docker/metadata-action@v3` should take care of the tagging for us.

Reply:
Yes, makes sense.